docker/Dockerfile.test

```diff
@@ -0,0 +1,34 @@
+FROM python:2.7.12
```

Comment:
Is this much different other than the CMD? Because we could just run the normal container with a different command or entrypoint.